src/gwproto/types/data_channel_gt.py

```diff
@@ -0,0 +1,39 @@
+"""Type data.channel.gt, version 001"""
+
+from typing import Literal, Optional
+
+from pydantic import BaseModel, ConfigDict, model_validator
+from typing_extensions import Self
+
+from gwproto.enums import TelemetryName
+from gwproto.property_format import (
+    LeftRightDotStr,
+    SpaceheatName,
+    UTCSeconds,
+    UUID4Str,
+)
+
+
+class DataChannelGt(BaseModel):
+    Name: SpaceheatName
+    DisplayName: str
+    AboutNodeName: SpaceheatName
+    CapturedByNodeName: SpaceheatName
+    TelemetryName: TelemetryName
+    TerminalAssetAlias: LeftRightDotStr
+    InPowerMetering: Optional[bool] = None
+    StartS: Optional[UTCSeconds] = None
+    Id: UUID4Str
+    TypeName: Literal["data.channel.gt"] = "data.channel.gt"
+    Version: Literal["001"] = "001"
+
+    model_config = ConfigDict(use_enum_values=True)
```

Comment:
I think this should go in a base class per comment [here](https://github.com/thegridelectric/gridworks-protocol/pull/346#issuecomment-2361921272).  That would require probably at least an hour of thought or experimentation, so we can also make that decision later.

Reply:
Maybe: 
class Gt(BaseModel):
    TypeName: Literal["gw"] = "gw"
    Version: Literal["000"] = "000"
    
    model_config = ConfigDict(use_enum_values=True)
    
    ? 
    
    I want to discuss with you before doing this.